google/gax/config.py

```diff
@@ -71,3 +71,7 @@ def exc_to_code(exc):
 
 create_stub = grpc.create_stub  # pylint: disable=invalid-name,
 """The function to use to create stubs."""
+
+
+ABORTION_ERRORS = (face.AbortionError, )
```

Comment:
The idea is that it should be possible

```python
import gax.config

gax.config.xxx  = someother_package.yyy  # simple references from another package
```

So this package should just contain assignments from grpc.py, with all the grpc specific import isolated to grpc.py

It's not really part of this CL, but much of the content of this module should be grpc.py:  sorry, I should have picked that up in earlier CLs.

Let's fix that starting with this change.

Move this to grpc.error, and and please use the more generic name API_ERRORS.